packages/editors/vim/after/syntax/ocaml/styled.vim

```diff
@@ -0,0 +1,20 @@
+if polyglot#init#is_disabled(expand('<sfile>:p'), 'css', 'after/syntax/ocaml/styled.vim')
+  finish
+endif
+
+if exists('b:current_syntax')
+  let s:current_syntax = b:current_syntax
+  unlet b:current_syntax
+endif
+
+let b:styled_nested_syntax = 1
+syn include @CssSyntax syntax/css.vim
+unlet b:styled_nested_syntax
+
+if exists('s:current_syntax')
+  let b:current_syntax = s:current_syntax
+endif
+
+syntax region styledPpxEncl matchgroup=ocamlPpxEncl start=+\[%cx+ end=+\]+ keepend contains=styledBody
+syntax region styledPpxEncl matchgroup=ocamlPpxEncl start=+\[%styled+ end=+\]+ keepend contains=styledBody
+syntax region styledBody start=+{|+ end=+|}+ contains=@CssSyntax contained
```

Comment:
Start can either be `{|` or `"`. It's a regular OCaml string, which doesn't matter what's the termination.

Reply:
Updated in the https://github.com/ahrefs/vim-styled-ppx